src/zoid/buttons/component.jsx

```diff
@@ -539,6 +536,7 @@ export const getButtonsComponent: () => ButtonsComponent = memoize(() => {
               fundingSource,
               fundingEligibility,
               enableFunding,
+              experiment,
```

Comment:
do we have unit tests we could add to that would assert this doesn't get removed in the future? simple regression test. although I guess more value here would be tests comparing first and second render.

Reply:
Yeah, I just realized we have Venmo standalone button tests that didn't catch that this was broken in the first place! I'll update this if we decide to move forward with these changes.

Thanks B 😄